git/filter_process_scanner_test.go

```diff
@@ -126,13 +127,13 @@ func TestFilterProcessScannerReadsRequestHeadersAndPayload(t *testing.T) {
 
 func TestFilterProcessScannerRejectsInvalidHeaderPackets(t *testing.T) {
 	from := bytes.NewBuffer([]byte{
-		0x30, 0x30, 0x30, 0x34, // 0004 (invalid packet length)
+		0x30, 0x30, 0x30, 0x33, // 0004 (invalid packet length)
```

Comment:
The comment should be `0003`.  Packet lengths include the 4-byte length field, so `0003` is not a valid packet (it would be a special one if it existed, but it doesn't).  We used to reject `0004`, but we no longer do, since they are valid packets, although sending them is unusual (since they're empty); this is just an error in not updating the comment.